HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 History
 =======
 
+0.1.8.1 (2022-09-29)
```

Comment:
Version numbers are major.minor.patch. I think we can just bump the patch version here.

```suggestion
0.1.9 (2022-09-29)
```

Reply:
noted and done